### PR TITLE
Added Box2D ChainShape render function

### PR DESCRIPTION
**Pull request by rubberbird**
Added Box2D ChainShape render function

### CODE REVIEW

**Comment by luanpotter** [comment]
Fixed the changelog conflict for you

File: lib/box2d/box2d_component.dart [hunk 1]
```diff
@@ -134,6 +134,31 @@ abstract class BodyComponent extends Component {
...
+    final List<Offset> points = [];
+    for (int i = 0; i < chainShape.getVertexCount(); i++) {
+      points.add(Offset(vertices[i].x, vertices[i].y));
+    }
+
+    renderChain(canvas, points);
+  }
+
+  void  renderChain(Canvas canvas, List<Offset> points) {
+      final Paint paint = Paint()
```
Comment (luanpotter) [inline lib/box2d/box2d_component.dart]: is there a way to override this colour? also, the default colour could be a static constant

**Comment by rubberbird** [comment]
Whats this build error? The command "set -e" exited with 0.
$ ./scripts/format.sh
files not formatted

File: lib/box2d/box2d_component.dart (hunk 1 above)
Comment (rubberbird) [reply lib/box2d/box2d_component.dart]: I just replicated the existing render functions. As it is just for viewing the bodies and a custom render function is required anyway, I figure this was ok. 

Example of existing below.

```
void renderCircle(Canvas canvas, Offset center, double radius) {
    final Paint paint = Paint()
      ..color = const Color.fromARGB(255, 255, 255, 255);
    canvas.drawCircle(center, radius, paint);
  }
```

**Comment by erickzanardo** [comment]
@rubberbird this is our script to verify if the files are formatted according our lint, to fix those formatting problems, just run `flutter format .` on the root of the project :)

**Comment by rubberbird** [comment]
> @rubberbird this is our script to verify if the files are formatted according our lint, to fix those formatting problems, just run `flutter format .` on the root of the project :)

Thats odd, as I ran that in my base project as you suggested. Are you able to give it a try for me?

File: lib/box2d/box2d_component.dart (hunk 1 above)
Comment (luanpotter) [reply lib/box2d/box2d_component.dart]: I see, makes sense

**Pull request merged**